src/Symfony/Component/Workflow/Tests/EventListener/AuditTrailListenerTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Symfony\Component\Workflow\Tests\EventListener;
+
+use Psr\Log\AbstractLogger;
+use Symfony\Component\EventDispatcher\EventDispatcher;
+use Symfony\Component\Workflow\Definition;
+use Symfony\Component\Workflow\EventListener\AuditTrailListener;
+use Symfony\Component\Workflow\MarkingStore\PropertyAccessorMarkingStore;
+use Symfony\Component\Workflow\Transition;
+use Symfony\Component\Workflow\Workflow;
+
+class AuditTrailListenerTest extends \PHPUnit_Framework_TestCase
+{
+    public function testItWorks()
+    {
+        $transitions = [
+            new Transition('t1', 'a', 'b'),
+            new Transition('t2', 'a', 'b'),
+        ];
+
+        $definition = new Definition(['a', 'b'], $transitions);
+
+        $object = new \stdClass();
+        $object->marking = null;
+
+        $logger = new Logger();
+
+        $ed = new EventDispatcher();
+        $ed->addSubscriber(new AuditTrailListener($logger));
+
+        $workflow = new Workflow($definition, new PropertyAccessorMarkingStore(), $ed);
+
+        $workflow->apply($object, 't1');
+
+        $expected = [
```

Comment:
@HeahDude please don't comment for this everywhere. fabbot.io will handle it on next push to the PR